schema/m3-region.schema.json

```diff
@@ -537,6 +537,74 @@
                 }
               }
             }
+          },
+          "enemies": {
+            "$id": "#/items/properties/enemies",
+            "type": "array",
+            "title": "Enemies; Things that try to kill you.",
+            "items": {
+              "$id": "#/items/properties/enemies/items",
+              "type": "object",
+              "title": "Enemies",
+              "required": [
+                "name",
+                "quantity",
+                "homeNodes"
```

Comment:
Is `homeNodes` really required since it seems mutually exclusive with `betweenNodes`?

Reply:
Seemed like better practice to always include homeNodes as a null. Should I remove them when null?

Reply:
The question is, do you think a null homeNodes brings value that omitting it doesn't?

A quick search suggests there might be a way to make them mutually exclusive. My suggestion is to try
```json
{ "required":
  [ 
    "name",
    "quantity"
  ]
},
"oneOf": [
  { "required":
    [ "homeNodes" ]
  },
  { "required":
    [ "betweenNodes" ]
  }
],
```